markdown/ch6.markdown

```diff
@@ -655,6 +655,39 @@ plot_connectome(x, y, C, labels=neuron_ids, types=neuron_types,
 
 <!-- exercise end -->
 
+## SIDEBOX: Solvers
```

Comment:
We should probably unify how we deal with these... Currently it's been block-quoted and had a `{.callout}` class annotation...